libnetwork/netavark/run.go

```diff
@@ -18,6 +19,18 @@ type netavarkOptions struct {
 	Networks map[string]*types.Network `json:"network_info"`
 }
 
+func (n *netavarkNetwork) Update(networkName string, networkDNSServers []string) error {
```

Comment:
this should be private and called from NetworkUpdate() I assume?

Reply:
I made it a separate call so its easier to write tests for libnetavark otherwise calling this function in tests creates a dependency on `netavark/aardvark` binary which is only updated on a release.

Reply:
But sure I can merge this with NetworkUpdate and `NetworkUpdate` can internally can `networkUpdate` and `execUpdate` and `libnetwork` tests can use `networkUpdate` instead of top level `NetworkUpdate`. What would you like to suggest.

Reply:
It cannot be made in two calls, this breaks locking.

If you want to test you can just pass `true` as netvark binary to the interface.